haas/api.py

```diff
@@ -84,6 +66,7 @@ def project_delete(project):
 
     If the project does not exist, a NotFoundError will be raised.
     """
+    get_auth_backend().require_admin()
```

Comment:
I don't have strong feelings on this, but do you think a user should be able to delete his/her own project?